### DISCUSSION TITLE
Team-level Pull Requests list

### DISCUSSION THREAD

**Discussion by ZachEspositoPCTY** [community/community#50525] [Pull Requests]
### Select Topic Area

Product Feedback

### Body

Hello,

As a senior dev on my team I need to be aware of all open PRs across all the repositories owned by my team. As far as I can tell there is no easy way to see all open PRs in a team at the moment. It would be dandy if there was something like a team-level Pull Requests page that could show this. Nav item might look like this:
![image](https://user-images.githubusercontent.com/110414185/226450134-974f8b6e-ee90-4376-b557-7f8ce582d598.png)

Thanks

**Comment by Neodevils** [comment]
You can try to watch Pull Request Events when it is happening. So you will be notified when there's a pull request.

**Reply by ZachEspositoPCTY** [reply]
True but I usually have to review them more than once so I'd like to be able to see all open PRs in a list because it would help me see what still needs attention

**Reply by Neodevils** [reply]
For solving your problem, you can see the PR's assigned to you with: 
![IMG_20230321_175045](https://user-images.githubusercontent.com/71941576/226644625-3acbfcec-ea0f-4349-9e0a-3f39f95e7a32.jpg)

So you can check all the details.

**Reply by ZachEspositoPCTY** [reply]
Maybe I'm doing something wrong but it seems like even when I'm assigned to a PR, if I review it then it no longer appears on that view in your screenshot, so that when the PR developer makes updates, it's hard for me to find the PR again

**Reply by Neodevils** [reply]
Hmm... I only know these things could help to you. Maybe you can try a webhook message showing up the new updates on your channel.

**Comment by bgianfo** [comment]
I think the only real workaround at the momen is to use https://github.com/dlvhdr/gh-dash or something like it and set the PR filter to be all your teams / repos organizations.

**Reply by voronoipotato** [reply]
This is not a workaround for people who can't use github api for security reasons, but thank you for helping.

**Comment by voronoipotato** [comment]
I don't understand why this wasn't included with the launch of a team. One of the purposes of a team is to track work. It has been an issue for major orgs for as long as teams have existed.  A team might want to be able to review the prs that a team made. Sometimes, more than one team. Currently your options are, "spam everyone on every team for every review" or "no ability to find any PRs for any team".   

There's a separate request for searching code by team, which is similarly upsetting. What is the utility of associating repos with a team if you can't even search them. You should be able to search by team for things if we have a team.

I'm not _even_ a manager and I noticed this is a problem. Managers have a binary choice between global spam making notifications useless, and flying blind. 

@queenofcorgis  Apologies, for the "@" but I'm having trouble searching if this request has been requested before. For example, searching "Team Pull" yields a bunch of random issues, while I do see some dupes of this specifically I can't really find the most upvoted one.

**Reply by voronoipotato** [reply]
(I said years, but it only has been as long as I've been working at my current job, it may have not been years)

**Reply by voronoipotato** [reply]
This is made worse by https://github.com/orgs/community/discussions/52113

**Reply by voronoipotato** [reply]
Also similar to https://github.com/orgs/community/discussions/48031

**Reply by voronoipotato** [reply]
I guess this is also related? https://github.com/orgs/community/discussions/46757

**Reply by Neodevils** [reply]
@voronoipotato, they can set up reviewers on pull requests templates with GitHub Actions. If there’s an event happening — such as a comment — they will be knowing it since they’re subscribed to the pull request.

**Reply by voronoipotato** [reply]
That's not particularly useful for observing work in progress.

**Reply by Neodevils** [reply]
Well, it is actually.

**Reply by voronoipotato** [reply]
It appears the draft prs don't show up for us in that case. Getting pinged for every PR isn't desirable. Also it appears to be a decent chunk of work to update the 200+ repos to support this.

**Reply by Neodevils** [reply]
If there is 200+ repositories; you would not check all of repositories at same time plus not like all members would work on it with collaborators.

**Reply by voronoipotato** [reply]
I'm having trouble understanding your perspective, in what workplace do you not want to know what your team is working on? I can understand for OSS/FOSS the labor is voluntary so it really does not matter, but for work it does matter at least a little bit. Remember all these features are available for an org, for larger orgs we don't want to be forced to create a new org for every team.
